src/utils.ts

```diff
@@ -58,8 +58,8 @@ export function createOriginHeaders (event: CompatibilityEvent, options: CorsOpt
     return { 'Access-Control-Allow-Origin': origin, Vary: 'Origin' }
   }
 
-  const requestOriginHeader = getRequestHeader(event, 'Origin') as string
```

Comment:
This is a bug of `getRequestHeader` function.
It returns `string | string[]` while it should return `string | string[] | undefined`.

I will create a bug report in the h3 upstream repository and add a reference to it as a comment.

---

Confession: I added the utility functions to the upstream, so it's my mistake.

Reply:
This conversation can be resolved once https://github.com/NozomuIkuta/h3-cors/pull/2#discussion_r966198831 is resolved.